apps/dapp-console-api/src/routes/apps/AppsRoute.ts

```diff
@@ -60,13 +60,23 @@ export class AppsRoute extends Route {
         throw Trpc.handleStatus(500, 'error fetching apps')
       })
 
+      const privyUser = await this.trpc.privy.getUser(privyDid).catch((err) => {
```

Comment:
We should be careful that we don't start to get rate limited by privy, we're basically doubling our calls to their API. If we run into any issues when a user hits the login endpoint we can pull the user their and save it to our db.